### DISCUSSION TITLE
Feature - Using UDP on Reverse Gateway

### DISCUSSION THREAD

**Discussion by 9to1url** [narrowlink/narrowlink#72] [Ideas]
That's be great NarrowLink already implemented p2p using QUIC, 😃

Could that UDP transport also implement on Gateway? I found many RTC applications need that. Thanks

**Comment by SajjadPourali** [comment]
> That's be great NarrowLink already implemented p2p using QUIC, 😃
> 
> Could that UDP transport also implement on Gateway? I found many RTC applications need that. Thanks

It is planned https://github.com/narrowlink/narrowlink/issues/16. But it's not a high priority at this time.
